start.gls.jobs.sh

```diff
@@ -20,17 +21,40 @@ row_count=$(bq --project_id="${project_id}" --format=csv query --use_legacy_sql=
 ############################################
 # as defined as queueSize in nextflow.config
 ############################################
-queue_size=4
-batches=$(( row_count / batch_size + 1 ))
-num_of_jobs=$(( concurrency / queue_size ))
+queue_size=100
```

Comment:
There is a bit repeated code between start.gls.jobs.sh and starrt.lsf.sh. I would imagine that similar logic would be needed for AWS batch as well. Is there anyway to consolidate the similar logics?

Reply:
Yes there maybe a way, I haven't seen the code for AWS batch yet. I can maybe merge them once that code is merged into master.

Reply:
Thanks in advance.